Formula/z/zsv.rb

```diff
@@ -0,0 +1,32 @@
+class Zsv < Formula
+  desc "Tabular data swiss-army knife CLI"
+  homepage "https://github.com/liquidaty/zsv"
+  url "https://github.com/liquidaty/zsv/archive/refs/tags/v1.0.0.tar.gz"
+  sha256 "218d184dabf847f97f6920dfa3ed4626f6816420109e3d15739151d7bfeca2ec"
+  license "MIT"
+  head "https://github.com/liquidaty/zsv.git", branch: "main"
+
+  depends_on "gcc" => :build
+  depends_on "jq"
+
+  uses_from_macos "ncurses"
+
+  fails_with :clang do
+    cause "GCC binaries have been observed to be more performant than clang!"
+  end
```

Comment:
that does not sound like a good reason?

Reply:
GCC is the recommended compiler as it has been observed to generate faster code compared to clang.
Any suggestions?

Reply:
Actually, it has been mentioned in the `zsv`'s [BUILD.md](https://github.com/liquidaty/zsv/blob/main/BUILD.md#a-note-on-compilers):

> GCC 11+ is the recommended compiler. Compared with clang, gcc in some cases seems to produce faster code for reasons we have not yet determined.

Reply:
- logged an upstream issue, https://github.com/liquidaty/zsv/issues/435